src/room/canonical_alias.rs

```diff
@@ -4,7 +4,12 @@ use ruma_events_macros::StateEventContent;
 use ruma_identifiers::RoomAliasId;
 use serde::{Deserialize, Serialize};
 
+use crate::StateEvent;
+
 /// Informs the room as to which alias is the canonical one.
+pub type CanonicalAliasEvent = StateEvent<CanonicalAliasEventContent>;
+
+/// The payload for `CanonicalAliasEvent`
```

Comment:
Just to keep consistent with the existing style, please add a dot at the end of each of these.

```suggestion
/// The payload for `CanonicalAliasEvent`.
```